lifecycle_manager/build_package.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
An alternative that would not require manually running this script is something like that:

```
data "archive_file" "lambda" {
  type              = "zip"
  source_file   = "main.py"
  output_path = "ec2-workerpool-lifecycle-manager.zip"
}
```

Reply:
good idea, done!